sdks/python/tox.ini

```diff
@@ -27,6 +27,7 @@ select = E3
 deps=
   pep8
   pylint
+  pyhamcrest
```

Comment:
Probably we should keep this dependency in setup.py as well to support 'python setup.py test' command. @charlesccychen to confirm.